components/contribution-flow/ContributionFlowStepContainer.js

```diff
@@ -137,7 +137,7 @@ class ContributionFlowStepContainer extends React.Component {
             showFeesOnTop={this.props.showFeesOnTop}
           />
         ) : (
-          <StepDetailsCrypto onChange={this.props.onChange} data={stepDetails} />
+          <StepDetailsCrypto onChange={this.props.onChange} data={stepDetails} currency={collective.currency} />
```

Comment:
Pass `collective` instead of `currency`.

Reply:
It will be later easier to read and understand what the value is supposed to be if it's written`collective.currency` rather than just `currency`.

Reply:
Thanks, makes sense. Corrected it. 👍🏽